mmdet/models/backbones/res2net.py

```diff
@@ -0,0 +1,324 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import build_conv_layer, build_norm_layer
+
+from ..builder import BACKBONES
+from .resnet import Bottleneck as _Bottleneck
+from .resnet import ResNet
+
+
+class Bottle2neck(_Bottleneck):
+    expansion = 4
+
+    def __init__(self,
+                 inplanes,
+                 planes,
+                 stride=1,
+                 dilation=1,
+                 downsample=None,
+                 style='pytorch',
+                 with_cp=False,
+                 conv_cfg=None,
+                 norm_cfg=dict(type='BN'),
+                 dcn=None,
+                 plugins=None,
+                 scale=4,
+                 base_width=26,
+                 stype='normal'):
```

Comment:
We may pack unused arguments in `kwargs` like in `ResNeXt`. Only `inplanes`, `planes`, `base_width`, `base_channles` are necessary.

Reply:
Done as suggested.